### PR TITLE
OTK install doc

### PR DESCRIPTION
**Pull request by APIM-KiranVaddadi**
**Description of the change**

**Benefits**

**Drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

**Checklist** 
- [] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [] Variables are documented in the README.md
- [] Title of the PR starts with chart name (e.g. `[charts/gateway]`)
- [] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

### CODE REVIEW

File: examples/otk/README.md [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+
+OTK installation involves
+1. Installation or upgrade of OTK solution kit.
+2. Installation or upgrade of OTK Database.
+3. Installation or upgrade of OTK customizations.
+
+| OTK Type | Database backed Gateway </br> (database.enabled=true) | Ephemeral Gateway </br> (database.enabled=false) |
+| ------------------------   | ---------------------- | ---------------------  |
+| Solution Kit Install/Upgrade | <ul><li>Uses post-install kubernetes Job (Headless installation - Restman)</li><li>Runs after Gateway startup</li></ul> |  <ul><li>Uses kubernetes init-container to bootstrap Gateway with OTK solution kits </li><li> Runs before start of Gateway</li><li> OTK Dual Gateway (DMZ/INTERNAL) configuration is not supported. </li></ul> |
+| OTK Database Install/upgrade </br> ***(NA for OTK type DMZ and OTK DB type - Cassandra)*** | <ul><li>Uses pre-install kubernetes Job (Lisqubase scripts)</li><li>Runs before Gateway startup</li></ul> | <ul><li>Uses pre-install kubernetes Job (Lisqubase scripts)</li><li>Runs before Gateway startup</li></ul> |
```
Comment (ab-sub) [inline examples/otk/README.md]: lets avoid NA . call it out as Not Applicable

Comment (ab-sub) [reply examples/otk/README.md]: please do a spell check. ex: Lisqubase

File: examples/otk/README.md [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+.....
+```
+
+### Customizations
+
+OTK customizations can be configured using Kubernetes config maps or secrets. Existing config maps and secrets can be configured as below.
+
+| Parameter                        | Description                               | Default                                                      |
+| -----------------------------    | -----------------------------------       | -----------------------------------------------------------  |
+| `otk.customizations.existingBundle.enabled` | Enable mounting existing configMaps/Secrets that contain OTK Bundles - see values.yaml for more info | `false`  |
```
Comment (ab-sub) [inline examples/otk/README.md:170]: should we talk about naming convention

File: examples/otk/README.md [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+| Customizations | <ul><li> Restman bundles applied using restman calls after Gateway startup which can be Kubernetes Config Maps and/or Secrets </li><li>Init containers - Bootstrapped on to Gateway</li></ul>  | <ul><li>Restman bundles bootstrapped on to gateway which can be Kubernetes Config Maps and/or Secrets</li><li> Init Containers - Bootstrapped on to Gateway</li></ul> |
+
+* [Quick Start](#quick-start)
+* [OTK with MySQL/Oracle database](#otk-with-mysql-or-oracle-database)
+* [OTK With Cassandra database](#otk-with-cassandra-database)
+* [Customizations](#customizations)
+
+### Quick Start
+
+OTK can be installed by okt.enabled=true. This will create OTK database using MySQL subchart and then bootstraps the OTK bundles on to the gateway (type SINGLE). The usage of MySQL subchart for OTK database do not represent production configuration.
```
Comment (ab-sub) [inline examples/otk/README.md]: okt.enabled (please update)

File: examples/otk/README.md [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+
+Add the layer7 repository:
+
+    helm repo add layer7 https://caapim.github.io/apim-charts/
+    helm repo update
+
+Then, you can install OTK by:
+
+    helm install my-ssg layer7/gateway --set-file "license.value=path/to/license.xml"  --set "otk.enabled=true" --set "license.accept=true"
```
Comment (ab-sub) [inline examples/otk/README.md]: please enclose within ``` so that the scroller is not necessary

File: examples/otk/README.md (hunk 1 above)
Comment (APIM-KiranVaddadi) [reply examples/otk/README.md]: fixed

File: examples/otk/README.md (hunk 4 above)
Comment (APIM-KiranVaddadi) [reply examples/otk/README.md]: updated.

File: examples/otk/README.md (hunk 3 above)
Comment (APIM-KiranVaddadi) [reply examples/otk/README.md]: corrected.

**Pull request merged**